src/capi/azext_capi/custom.py

```diff
@@ -129,25 +130,34 @@ def find_kubectl_current_context():
         output = run_shell_command(command)
         output = output.strip()
         return output
-    except subprocess.CalledProcessError:
-        return None
+    except subprocess.CalledProcessError as err:
+        if "current-context is not set" in err.stdout:
```

Comment:
`return None` is the default behavior at the end of a python function, so this could be just:

```python
if "current-context is not set" not in err.stdout:
    raise
```